src/Arcus.Templates.AzureFunctions.ServiceBus.Topic/OrderFunction.cs

```diff
@@ -51,6 +59,45 @@ public OrderFunction(IAzureServiceBusMessageRouter messageRouter)
             AzureServiceBusMessageContext messageContext = message.GetMessageContext(_jobId);
             MessageCorrelationInfo correlationInfo = message.GetCorrelationInfo();
             await _messageRouter.RouteMessageAsync(message, messageContext, correlationInfo, cancellationToken);
+        } 
+#endif
+#if Isolated
+        /// <summary>
+        /// Process an Azure Service Bus topic <paramref name="messageBody"/> as an <see cref="Order"/>.
+        /// </summary>
+        /// <param name="messageBody">The incoming message on the Azure Service Bus topic, representing an <see cref="Order"/>.</param>
+        /// <param name="context">The execution context for this Azure Functions instance.</param>
+        [Function("order-processing")]
+        public async Task Run(
+            [ServiceBusTrigger("order-topic", "order-subscription", Connection = "ServiceBusConnectionString")] byte[] messageBody,
```

Comment:
Just wondering why we receive a byte array here, instead of a ServiceBusReceivedMessage, like we have in the in-process version ?

Reply:
Because that is not supported. The isolated variant only allows access to the message's contents and via via a limited access to some (but not all) metadata.
https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-service-bus-trigger?tabs=isolated-process%2Cextensionv5&pivots=programming-language-csharp#message-metadata